File: week 1/day 1/algo-99-bottles/99_bottles.py
def bottle_song():
    # write your code here!
    bottles = 99
    arr = []

    while (bottles > 1):
        string = f'{bottles} bottles of beer on the wall, {bottles} of beer. Take one down and pass it around, {bottles-1} bottles of beer on the wall.'
        arr.append(string)
        bottles -= 1


bottle_song()

File: week-01/day-01/algo-99-bottles/99_bottles.js
function bottleSong() {
  // Write your code here!
  // init variables
  let bottles = 99;
  let arr = [];
  for (bottles; bottles > 1; bottles--) {
    let str = `
    ${bottles} bottles of beer on the wall, ${bottles}  bottles of beer.
    Take one down and pass it around, ${bottles - 1} ${
      bottles === 2 ? 'bottle' : 'bottles'
    } of beer on the wall.`;

    arr = [...arr, str];
  }
  let ending = `
  1 bottle of beer on the wall, 1 bottle of beer.
  Take one down and pass it around, no more bottles of beer on the wall.
  No more bottles of beer on the wall, no more bottles of beer.
  Go to the store and buy some more, 99 bottles of beer on the wall.`;
  arr.push(ending);
  console.log(arr.join('\n'));
}
bottleSong();
